src/Core/EnvironmentSettings.cs

```diff
@@ -25,14 +25,36 @@ namespace SonarLint.VisualStudio.Core
     public class EnvironmentSettings : IEnvironmentSettings
     {
         internal const string TreatBlockerAsErrorEnvVar = "SONAR_INTERNAL_TREAT_BLOCKER_AS_ERROR";
+        internal const string CFamilyAnalysisTimeoutEnvVar = "SONAR_INTERNAL_CFAMILY_ANALYSIS_TIMEOUT_MS";
+        public const string SonarLintDownloadUrlEnvVar = "SONARLINT_DAEMON_DOWNLOAD_URL";
```

Comment:
The name of this environment variable is a in slightly different format to the INTERNAL ones, but it would be a breaking change to change it now.